tests/core/conftest.py

```diff
@@ -58,19 +64,35 @@ def admin_mc():
     }, verify=False)
     client = rancher.Client(url=BASE_URL, token=r.json()['token'],
                             verify=False)
-    return ManagementContext(client)
+    k8s_client = kubernetes_api_client(client, 'local')
+    return ManagementContext(client, k8s_client)
+
+
+def kubernetes_api_client(rancher_client, cluster_name):
+    c = rancher_client.by_id_cluster(cluster_name)
+    kc = c.generateKubeconfig()
+    loader = KubeConfigLoader(config_dict=yaml.load(kc.config))
+    client_configuration = type.__call__(Configuration)
+    loader.load_and_set(client_configuration)
+    k8s_client = ApiClient(configuration=client_configuration)
+    return k8s_client
 
 
 @pytest.fixture
 def admin_cc(admin_mc):
     """Returns a ClusterContext for the local cluster for the default global
     admin user."""
-    cluster = admin_mc.client.by_id_cluster('local')
+    cluster, client = cluster_and_client('local', admin_mc.client)
```

Comment:
`local` should end up as a const, we use it too much then we can set it later with an env var

Reply:
lets address this one when we integrate with hussein's work to pull in secondary cluster